mempool/metrics.go

```diff
@@ -22,7 +22,8 @@ type Metrics struct {
 	// Histogram of transaction sizes, in bytes.
 	TxSizeBytes metrics.Histogram
 
-	// Number of failed transactions.
+	// Number of failed transactions. These were marked invalid by the application
+	// in either check tx or recheck tx
```

Comment:
```suggestion
	// FailedTxs defines the number of failed transactions. These were marked
	// invalid by the application in either CheckTx or RecheckTx.
```

Proposal to start the comment with `FailedTxs` to conform to the other fields in this struct and because

>  doc comments for types start with complete sentences naming the declared symbol

[source](https://tip.golang.org/doc/comment)